Vagrantfile

```diff
@@ -160,6 +160,7 @@ end
 VAGRANTFILE_API_VERSION = "2"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.box_check_update = false
+    config.vbguest.auto_update = false
```

Comment:
netplugin box has guest additions already. the machine doesnt need to have guest additions right ?  Its needed only if the box doesnt have guest additions installed.